libraries/AP_HAL_ChibiOS/hwdef/common/stm32_util.h

```diff
@@ -54,8 +54,8 @@ uint32_t get_fattime(void);
 
 enum rtc_boot_magic {
     RTC_BOOT_OFF  = 0,
-    RTC_BOOT_HOLD = 0xb0070001,
-    RTC_BOOT_FAST = 0xb0070002
```

Comment:
I deliberately made them different as the behavior is not the same. When I tested it I found that the behaviour in the px4 bootloader is really quite strange.